app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,47 @@
+
+<nav class="navbar navbar-default ">
+  <div class="container-fluid navigation">
+    <div class="navbar-left navigation-left">
+      <h2><%= link_to 'FACEBOOK', root_path %></h2>
+    </div>
+    <%if user_signed_in?%>
+      <ul class="nav navbar-nav navbar-right header">
+       <li><%= link_to "Post", "#" %></li>
```

Comment:
I clicked here thinking it'll take me to `posts#index` view. What do you plan on using this link for when it's finally functional?